lxqt-config-monitor/main.cpp

```diff
@@ -52,7 +52,7 @@ int main(int argc, char** argv)
     // Command line options
     QCommandLineParser parser;
     QCommandLineOption loadOption(QStringList() << "l" << "loadlast",
-            app.tr("Load last settings."));
+            LXQt::SingleApplication::tr("Load last settings."));
```

Comment:
Yes, without it I get the following (lupdate version 5.5.1):
/home/src/lxqt-config/lxqt-config-monitor/main.cpp:55: tr() cannot be called without context